tests/distributed/test_remote_peas/test_remote_peas.py

```diff
@@ -270,9 +275,9 @@ async def test_psuedo_remote_peas_shards(gateway, head, worker, polling):
         worker_host = __docker_host__ if is_remote(head) else HOST
```

Comment:
I think its never correct to use __docker_host__ here no? The head is not running as a container as far as I understand so __docker_host__ will never be resolved correctly? You should use that rather to communicate from inside a container to the host